File: Unidad 3/Java/Programa7U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa7u3;

import java.util.Scanner;
public class Programa7U3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int temp;
        System.out.println("Programa para saber la temperatura actual");
        System.out.println("Introdusca la temperatura");
        temp = sc.nextInt();
        
        if(temp>27 ){
            System.out.println("Hace calor");
        }else if(temp >20 && temp < 27){
            System.out.println("Clima agradable");
        }else if(temp < 20){
            System.out.println("Clima fresco");
        }
        
    }
}

File: Unidad 3/C#/Programa7U3.cs
// See https://aka.ms/new-console-template for more information

using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class Programa7U3
    {
        static void Main(string[] args)
        {
            int temp;
            Console.WriteLine("Programa para saber la temperatura actual");
            Console.WriteLine("Introdusca la temperatura");
            temp = Convert.ToInt32(Console.ReadLine());

            if (temp > 27)
            {
                Console.WriteLine("Hace calor");
            }
            else if (temp > 20 && temp < 27)
            {
                Console.WriteLine("Clima agradable");
            }
            else if (temp < 20)
            {
                Console.WriteLine("Clima fresco");
            }
        }
    }
}